modules/ixBidAdapter.js

```diff
@@ -76,7 +75,8 @@ const SOURCE_RTI_MAPPING = {
   'epsilon.com': '', // Publisher Link, publinkId
   'audigent.com': '', // Hadron ID from Audigent, hadronId
   'pubcid.org': '', // SharedID, pubcid
-  'trustpid.com': '' // Trustpid
+  'trustpid.com': '', // Trustpid
```

Comment:
this is going to make a conflict with the prebid 8 version of your adapter, could you pr that as well? Trustpid renamed themselves